openshift/fluent.conf

```diff
@@ -49,25 +49,36 @@
   capitalize_regex_backreference yes
   <rule>
     key     code
-    pattern ^4\d\d$
+    # filter for 5** HTTP response codes
+    pattern ^5\d\d$
     tag     ${tag}.error
   </rule>
+
+  <rule>
+    key     log
+    # filter logs containing string 'EHOSTUNREACH'. for example: 'ERR! Error: connect EHOSTUNREACH 172.50.242.155:6379'
```

Comment:
We should avoid posting actual IP addresses here, even if they are just examples. Change it to a different fake IP like `127.0.0.1' or something.